src/content/quickstart/verify_with_discovery.md

```diff
@@ -1,15 +1,37 @@
 #### Verify Deployment
 
-To verify the deployment follow the steps below which creates an nginx service and
+To verify the deployment follow the steps below which creates a headless or ClusterIp nginx service and
 ServiceExport for it.
 
+```bash
+export KUBECONFIG=cluster-b/auth/kubeconfig
+```
+
+##### Deploy ClusterIP Service
+
+To verify a ClusterIP service follow the steps below
```

Comment:
I think this should say Deploy but I don't think we need this statement as its redundant with the section title...

Reply:
Removed it.